pkg/apis/infinispan/v1/types_util.go

```diff
@@ -24,6 +27,21 @@ const (
 	ImageTypeNative ImageType = "Native"
 )
 
+const (
+	// PodTargetLabels labels propagated to pods
+	PodTargetLabels string = "infinispan.org/podTargetLabels"
+	// TargetLabels labels propagated to services/ingresses/routes
+	TargetLabels string = "infinispan.org/targetLabels"
```

Comment:
WDYT to using `infinispan.org/targetLabels` for labels that should be applied to both pods and services?

We then introduce `infinispan.org/svcTargetLabels` for services/ingresses/routes.

Reply:
we followed the Prometheus approach for the naming